### ISSUE TITLE
[2.7.0rc2] Exception after using select random user function

### ISSUE THREAD

**Issue by schrd** [bigbluebutton/bigbluebutton#18689]
**Describe the bug**
An error is logged under some weird conditions if the select random user feature is used.

~~~
[16:00:38:0429] ERROR: clientLogger: 
Issue in Random User Select container caused by back-end crash
Value of 6 randomly selected users was passed as {}
However, it is handled.
Error message:
TypeError: r[0] is undefined
~~~

**To Reproduce**

This is not 100% reproducible, sometimes it is required to switch more often between the users and randomly select users

1. Start a meeting
2. join a second user
3. make second user presenter
4. make first user presenter
5. use select random user function
6. repeat until an error message is logged in the browser console.

**BBB version:**
2.7.0rc2

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser Firefox user 1, Chrome User 2
 - Version 102ESR, 116

**Comment by antobinary** [comment]
Thank you for reporting!
We were able to reproduce this but we don't see any symptoms of this error (i.e. if we had the dev tools closed we would not detect it). Could you please confirm if this was disruptive in any way or was also not noticeable?

**Comment by schrd** [comment]
No, there were no obvious symptoms or disruptions.

**Comment by antobinary** [comment]
Closing as this won't be patched in BBB 2.7 and in BBB 3.0 we've reworked the randomly select user (and it's now a plugin -- https://github.com/bigbluebutton/plugins/tree/main/pick-random-user-plugin )
